device/dhcp/clients/v6/dhcpv6client.yaml

```diff
@@ -0,0 +1,146 @@
+components:
+  schemas:
+    Device.Dhcpv6client:
+      description: >-
+        Configuration for emulated DHCPv6 Client on a single Interface.
+        https://www.rfc-editor.org/rfc/rfc8415.html
+      type: object
+      x-status:
+        status: under_review
+      required: [name]
+      properties:
+        name:
+          x-include: ../../../../common/common.yaml#/components/schemas/Named.Object/properties/name
+          x-field-uid: 1
+        rapid_commit:
+          description: >-
+            If Rapid Commit is set, client initiates Rapid Commit two-message exchange by including Rapid Commit option 
+            in Solicit message.
+          type: boolean
+          default: false
+          x-field-uid: 2
+        ia_type:
+          $ref: '#/components/schemas/Device.Dhcpv6client.IaType'
+          x-field-uid: 3
+        duid_type:
+          $ref: '#/components/schemas/Device.Dhcpv6client.DuidType'
+          x-field-uid: 4
+
+    Device.Dhcpv6client.IaType:
+      type: object
+      properties:
+        choice:
+          description: >-
+            Identity Association: a collection of leases assigned to a client. Each IA has an associated IAID. 
```

Comment:
Might be good to describe the difference between the options in terms of protocol emulation , esp. since iana/iapd and ianapd refer to the same structure with same fields.

Reply:
updated